scripts/templates/create-component/EmptyTypes.mustache

```diff
@@ -1,5 +1,5 @@
+import { ITextSlot } from 'office-ui-fabric-react';
```

Comment:
Importing from the root of OUFR will probably cause errors with lint-imports.

Reply:
I've also seen others mention that this is ok to do with tree shaking and that we should do this instead of deep imports. What would you suggest be the actual import? I worry that people actually have conflicting stances on this and have wished we'd make an import standards page for some time...

Reply:
In projects where lint-imports is enabled, it will give an error on imports which don't reference physical files (because they cause errors with AMD bundling). I think `office-ui-fabric-react/lib/index` is the closest alternative which won't cause errors with AMD? But I totally agree we should have a larger discussion (after v7 and //build...) about "proper" import styles.